migrations/1668051174693_add-new-column-to-track-number-of-times-a-post-has-been-flagged.js

```diff
@@ -0,0 +1,10 @@
+exports.shorthands = undefined
```

Comment:
didn't take note of the migration name in relation to the `likesCount` column we added. The current filename seems best suited as a comment in the file itself to give more context on the field which is good.

However, I think the filename should be related to the specific change happening instead of the reason. So it can just be `add-flagsCount-column-to-posts-table`

Reply:
yes please